pages/css/architecture.md

```diff
@@ -50,46 +56,10 @@ lib/neat.scss
 _overrides.scss
 ```
 
-For the `util`, `typography`, `elements`, and `overrides` files, once they grow
-too large (300 lines or more) in size, split them into their own folder with
-sub files.
-
-```sh
-elements/_all.scss
-elements/_p.scss
-elements/_h.scss
-typography/_body.scss
-typography/_links.scss
-overrides/_breakpoints.scss
-overrides/_util.scss
-util/_center.scss
-util/_clearfix.scss
-```
-
-
-## Importing
-As you likely know, CSS rules that are later in the file override earlier
-rules. This means Sass imports can be used to control inheritance and
-specificity.
-
-- Start with base elements.
-- Move to single nested classes and utils.
-- Move next to more specific classes, often with nesting.
-- Move next to overrides, possibly with !important rules.
-- Import alphabetically.
-- Only modify import order for groups of files, not specific files.
+## Atomic architecture
```

Comment:
Help, I don't know what to write about atomic CSS because I can't stop hating it long enough to figure it out. :-p We should do some user interviews with folks like @hursey013 who have used/appreciated atomic CSS to figure out what they find it useful for.

Reply:
Can't help you. I think it's only suitable for prototypes.